src/app/archive/lib/processor.ml

```diff
@@ -4728,12 +4728,26 @@ let run pool reader ~genesis_constants ~constraint_constants ~logger
         Deferred.unit )
 
 (* [add_genesis_accounts] is called when starting the archive process *)
-let add_genesis_accounts ~logger
-    ~(precomputed_values_opt : Precomputed_values.t option) pool =
-  match precomputed_values_opt with
+let add_genesis_accounts ~logger ~(runtime_config_opt : Runtime_config.t option)
```

Comment:
*reviewer note*

now we use the runtime config opt instead of the precomputed_value type. This seems a like we want to expose more to deeper levels of the application. The intention is to now parse the precomputed values out.

Reply:
now we pass in the constraint and genesis constants from a higher level.

Reply:
i was wrong we now compute the genesis proof and inputs within the add genesis accounts function.